cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java

```diff
@@ -782,20 +782,6 @@ public PodDisruptionBudget generatePodDisruptionBudget(boolean stableIdentities)
         }
     }
 
```

Comment:
No need for the double empty lines here which happen after you delete the method.